scripts/pi-hole/js/footer.js

```diff
@@ -133,7 +133,7 @@ function initCheckboxRadioStyle() {
 
   function applyCheckboxRadioStyle(style) {
     boxsheet.attr("href", getCheckboxURL(style));
-    var sel = $("input[type='radio'],input[type='checkbox']");
+    var sel = $("input[type='radio'],input[type='checkbox']").not("#selSec").not("#selMin");
```

Comment:
Can you add a comment above this? Otherwise, in a few months, we will have forgotten what `selSec` and `selMin` in and why it is excluded here.

Reply:
```suggestion
    // Get all radio/checkboxes for theming, with the exception of the two radio buttons on the custom disable timer
    var sel = $("input[type='radio'],input[type='checkbox']").not("#selSec").not("#selMin");
```

Reply:
Ultimate lazy way to add a change.